include/crpropa/PhotonBackground.h

```diff
@@ -2,30 +2,50 @@
 #define CRPROPA_PHOTONBACKGROUND_H
 
 #include "crpropa/Common.h"
+#include "crpropa/Referenced.h"
 
 #include <string>
 
 namespace crpropa {
-
 /**
- * \addtogroup EnergyLosses
+ * \addtogroup PhotonFields
```

Comment:
As you added the new doxygen group, you should also add appropriate documentation in the new system, i.e. one file buildingblocks/photonfields.rst and also adding the new file to building_blocks.rst.